packages/hw-app-eth/src/contracts.ts

```diff
@@ -36,21 +36,13 @@ export const loadInfosForContractMethod = async (
     ...defaultPluginsLoadConfig,
     ...userPluginsLoadConfig,
   };
+
   let data = !baseURL
     ? {}
     : await axios
         .get(`${baseURL}/plugins/ethereum.json`)
         .then((r) => r.data as any)
-        .catch((e) => {
-          if (
-            e.response &&
-            400 <= e.response.status &&
-            e.response.status < 500
-          ) {
-            return null; // not found cases can be ignored to allow future changes in endpoint without failing a signature to be done.
-          }
-          throw e;
-        });
+        .catch(() => null);
```

Comment:
not a good idea to hide all possible error cases. but maybe there is a reason?

Reply:
Since this API will be used by everyone (including external wallets), if we throw we will just cancel the transaction. The idea here to not make our backend become a single point of failure for all external wallets ^^

Reply:
to be defined what shall be the behavior but it could be a "network is down" or a "undefined is not a function" (e.G. bug in axios) and in these 2 cases, the error should be thrown because it make a flawky support of a feature 🤔

Reply:
i'll keep your change and add a log